### ISSUE TITLE
I'm in

### ISSUE THREAD

**Issue by shardcore** [dariusk/NaNoGenMo#30]
Is the intention to 'publish' them in any form? e.g. amazon

Can they be (algo) illustrated?

**Comment by dariusk** [comment]
There is literally no prescription as to what these books may contain, nor how they are generated, except that there must be 50k words. "Words" intentionally left vague.

As for publishing, who knows! I claim no ownership over anything produced for this, but towards the end it's entirely possible that I'd put a call out to people who might want to do an anthology or something? Who knows.

**Comment by MichaelPaulukonis** [comment]
An "anthology" of 50K+ word books? More like a librarythology, amirite?

Or a shelf, I suppose.

**Comment by dariusk** [comment]
Yeah hence "or something" -- it would probably be an anthology of interesting portions of code, illustrated with examples of output. At least that's what I'm thinking.

**Comment by catseye** [comment]
Some evil thoughts regarding illustration:
1. A graphic novel is a kind of novel!
2. A picture is worth a thousand words!

**Comment by wordsmythe** [comment]
I sense a SlideFree "novel" coming on.

**Comment by wordsmythe** [comment]
I mean OutSlide.
